replace-default-plugin.js

```diff
@@ -6,7 +6,10 @@ module.exports = function (config, helpers, newPlugin, esmPlugin) {
       config.plugins.splice(index, 1, newPlugin);
     }
   });
-  if (helpers.getPluginsByName(config, 'BabelEsmPlugin')) {
+  if (pluginInstances.length === 0) {
```

Comment:
I had to add this for the service worker file to get generated on the other end, looks like `SWPrecacheWebpackPlugin` is missing so it never replaces it

Reply:
wait! what cli version are you using it?